validator/OWNERS

```diff
@@ -7,8 +7,8 @@
       owners: [{name: 'ampproject/wg-caching', required: true}],
     },
     {
-      pattern: '{package.json,package-lock.json}',
-      owners: [{name: 'ampproject/wg-caching', requestReviews: false}],
+      pattern: '{package.json,package-lock.json,WORKSPACE}',
+      owners: [{name: 'ampproject/wg-caching', required: false}],
```

Comment:
This overrides the `required: true` on line 7 (only for `package.json`, `package-lock.json`, and `WORKSPACE`).